spark3/src/main/java/org/apache/iceberg/spark/SparkCatalog.java

```diff
@@ -400,6 +404,7 @@ public final void initialize(String name, CaseInsensitiveStringMap options) {
             .toArray(new String[0]);
       }
     }
+    this.purgeDataAndMetadata = options.getBoolean(PURGE_DATA_AND_METADATA, PURGE_DATA_AND_METADATA_DEFAULT);
```

Comment:
We now have a `gc.enabled` property for the similar behavior ( https://github.com/apache/iceberg/pull/1796), could we introduce the existing property to this spark catalog ?  Sounds like the `gc.enabled=false` will keep all the data files but still remove the metadata files while what you expected is keep both the data files and metafiles..

Reply:
We can use this property in Spark Catalog, but from my point of view it is better to have additional property. 
But I can change it